types/evidence.go

```diff
@@ -1770,3 +1775,10 @@ func NewMockPOLC(height int64, time time.Time, pubKey crypto.PubKey) ProofOfLock
 		PubKey: pubKey,
 	}
 }
+
+func maxTime(t1 time.Time, t2 time.Time) time.Time {
```

Comment:
Should this be added to time pkg in libs?

Reply:
not until it's used (copied) in multiple packages